civic_europe/templates/formtools/wizard/wizard_form.html

```diff
@@ -7,6 +7,9 @@
 <div class="wizard-page">
     <form class="wizard custom-form container-narrow" method="post" enctype="multipart/form-data" novalidate>
         <h1>{{ view.title }} | {{ form.section_name }}</h1>
+        {% if form.section_description_header  %}
+        <b>{{ form.section_description_header }}</b>
```

Comment:
@phillimorland Do we use the bold-tag? Or do we only style stuff to be bold and would use p here? Or span? And add a a class to style it? I guess there is a bootstrap utility-class?!

Reply:
we only use bold tags in the emails, here I would whack it in a span and then use the font-weight-bold class